internal/workflow/processing_test.go

```diff
@@ -1631,3 +1637,99 @@ func (s *ProcessingWorkflowTestSuite) TestFailedPIPAM() {
 	s.True(s.env.IsWorkflowCompleted())
 	s.Error(s.env.GetWorkflowResult(nil))
 }
+
+func (s *ProcessingWorkflowTestSuite) TestInternalUpload() {
+	cfg := config.Configuration{
+		A3m:          a3m.Config{ShareDir: s.CreateTransferDir()},
+		Preservation: pres.Config{TaskQueue: temporal.A3mWorkerTaskQueue},
+		Storage:      storage.Config{DefaultPermanentLocationID: locationID},
+	}
+	s.SetupWorkflowTest(cfg)
+
+	sipName := "name.zip"
```

Comment:
Nitpick: I know the actual text is irrelevant but it seems weird that a SIP name would have a "zip" extension. Maybe just "name" would be clearer?